specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2025-04-01/containerregistry.json

```diff
@@ -3366,7 +3338,8 @@
           "type": "string"
         },
         "syncProperties": {
-          "$ref": "#/definitions/SyncProperties"
+          "$ref": "#/definitions/SyncProperties",
+          "description": "The sync properties of the connected registry with its parent."
```

Comment:
Is the `description` auto-generated? I don't see other places have both `$ref` and `description` properties.

Reply:
Not auto generated. Copied from previous swagger.